wolfcrypt/src/tfm.c

```diff
@@ -2158,6 +2251,17 @@ static int _fp_exptmod_ct(fp_int * G, fp_int * X, int digits, fp_int * P,
 #ifdef WOLFSSL_SMALL_STACK
    XFREE(R, NULL, DYNAMIC_TYPE_BIGINT);
 #endif
+
+#if defined(DEBUG_WOLFSSL) && defined(WOLFSSL_ESPIDF)
+    /* TODO consider moving / removing this */
```

Comment:
Perhaps remove this? Not adding much value.

Reply:
Agreed. I address this now in the [esp_clean_result()](https://github.com/gojimmypi/wolfssl/blob/d0756bd406518b7e6ad812d1c076df88d7b6c474/wolfcrypt/src/port/Espressif/esp32_mp.c#L460).